solutions/devsprint-matheus-reis-1/RealEstateApp/UI/Screens/PropertyDetails/Components/PropertyInfoCollectionViewCell.swift

```diff
@@ -0,0 +1,149 @@
+        areaLabel.textAlignment = .left
+        areaLabel.font = .systemFont(ofSize: 14)
+        areaLabel.textColor = .gray
+        return areaLabel
+    }()
+
+    public lazy var bedroomsLabel: UILabel = {
+        let bedroomsLabel = UILabel()
+        bedroomsLabel.translatesAutoresizingMaskIntoConstraints = false
+        bedroomsLabel.textAlignment = .left
+        bedroomsLabel.font = .systemFont(ofSize: 14)
+        bedroomsLabel.textColor = .gray
+        return bedroomsLabel
+    }()
+
+    public lazy var bathroomsLabel: UILabel = {
+        let bathroomsLabel = UILabel()
+        bathroomsLabel.translatesAutoresizingMaskIntoConstraints = false
+        bathroomsLabel.textAlignment = .left
+        bathroomsLabel.font = .systemFont(ofSize: 14)
+        bathroomsLabel.textColor = .gray
+        return bathroomsLabel
+    }()
+
+    public lazy var garageLabel: UILabel = {
+        let garageLabel = UILabel()
+        garageLabel.translatesAutoresizingMaskIntoConstraints = false
+        garageLabel.textAlignment = .left
+        garageLabel.font = .systemFont(ofSize: 14)
+        garageLabel.textColor = .gray
+        return garageLabel
+    }()
+
+    public lazy var addressLabel: UILabel = {
+        let addressLabel = UILabel()
+        addressLabel.translatesAutoresizingMaskIntoConstraints = false
+        addressLabel.textAlignment = .left
+        addressLabel.font = .systemFont(ofSize: 14)
+        addressLabel.textColor = .gray
+        return addressLabel
+    }()
+
+func configureSubviews() {
+    addSubview(priceLabel)
+    addSubview(condominiumFeeLabel)
+    addSubview(urbanTaxLabel)
+    addSubview(areaLabel)
+    addSubview(bedroomsLabel)
+    addSubview(bathroomsLabel)
+    addSubview(garageLabel)
+    addSubview(addressLabel)
+}
+
+func configureSubviewsConstraints() {
+   NSLayoutConstraint.activate([
+        priceLabel.leftAnchor.constraint(equalTo: self.leftAnchor),
+        priceLabel.topAnchor.constraint(equalTo: self.topAnchor)
+   ])
+    
+    NSLayoutConstraint.activate([
+        condominiumFeeLabel.topAnchor.constraint(equalToSystemSpacingBelow: priceLabel.bottomAnchor, multiplier: 0.7),
+        condominiumFeeLabel.leftAnchor.constraint(equalTo: priceLabel.leftAnchor)
+    ])
+    
+    NSLayoutConstraint.activate([
+        urbanTaxLabel.leftAnchor.constraint(equalToSystemSpacingAfter: condominiumFeeLabel.rightAnchor, multiplier: 3),
+        urbanTaxLabel.topAnchor.constraint(equalTo: condominiumFeeLabel.topAnchor)
+    ])
+    
+    NSLayoutConstraint.activate([
+        areaLabel.topAnchor.constraint(equalTo: condominiumFeeLabel.bottomAnchor, constant: 3),
+        areaLabel.leftAnchor.constraint(equalTo: condominiumFeeLabel.leftAnchor)
+    ])
+    
+    NSLayoutConstraint.activate([
+        bedroomsLabel.topAnchor.constraint(equalTo: areaLabel.topAnchor),
+        bedroomsLabel.leftAnchor.constraint(equalToSystemSpacingAfter: areaLabel.rightAnchor, multiplier: 0.5)
```

Comment:
Acredito que o Matheus consiga confirmar isso hoje, mas pelo que eu andei lendo, é interessante sempre tentarmos usar leadingAnchor e trailingAnchor no lugar de leftAnchor e rightAnchor, respectivamente.
As propriedades Leading e Trailing são afetadas pelo local do dispositivo, enquanto que Left  e Right seriam absolutos.
Em países onde a leitura é feita da esqueda para a direita ( que é o nosso caso ), realmente não faria diferença usarmos left ou leading, por exemplo. 
Mas em países onde a leitura é feita da direita para a esquerda ( Arabic, por exemplo ), o leading seria o lado direito ( onde a pessoa começaria a leitura ) e o trailing seria a parte esquerda ( onde finalizaria ), sendo então invertidos.

```suggestion
        bedroomsLabel.leadingAnchor.constraint(equalToSystemSpacingAfter: areaLabel.trailingAnchor, multiplier: 0.5)
```

Reply:
Você tem razão, @tandersonf ! Inclusive, imagino que seu argumento tenha saído diretamente [deste artigo](https://www.hackingwithswift.com/example-code/uikit/whats-the-difference-between-leading-trailing-left-and-right-anchors). Usar `leftAnchor`/`rightAnchor` vai forçar que seus elementos estejam fixos na esquerda/direita. O ideal é usar sempre `leadingAnchor` e `trailingAnchor` pra garantir que suas telas tenham o comportamento adequado pra cada caso.